app/controllers/all_casa_admins_controller.rb

```diff
@@ -25,7 +26,7 @@ def update
     @user = current_all_casa_admin
 
     if @user.update(all_casa_admin_params)
-      flash[:success] = "Profile was successfully updated."
+      flash[:success] = 'Profile was successfully updated.'
```

Comment:
Hey, nice work. Can you please use this part of the README 🙂: 

Cleaning up before you pull request

bundle exec standardrb --fix auto-fix Ruby linting issues more linter info
bundle exec erblint --lint-all --autocorrect ERB linter
yarn lint:fix to run the JS linter and fix isses
rake factory_bot:lint if you have been editing factories and want to find factories and traits which produce invalid objects

This way we all have an unified code base.

Reply:
Thank so much! I did realize that.